### ISSUE TITLE
Error thrown when totalCount is only field on connection field in 2.0.0-alpha.3

### ISSUE THREAD

**Issue by dmoree** [neo4j/graphql#332]
**Describe the bug**
When querying `totalCount` on a connection field an error `"Cannot read property 'fieldsByTypeName' of undefined"` is thrown if it is the only field.

**Type definitions**
Based on `neo-push` example

**To Reproduce**

1. Clone issue repository
```
git clone https://github.com/dmoree/neo4j-graphql-issue-total-count.git
```
2. Seed database and start server
```
cd neo4j-graphql-issue-total-count && yarn install && yarn seed && yarn start
```
3. run following `query` in GraphiQL at `http://localhost:4000/graphql` (or as defined in `.env`)
```graphql
query TotalCountOfComments {
  posts {
    commentsConnection {
      totalCount
    }
  }
}

```

**Expected behavior**
Would expect `totalCount: 3` as seen in the following `query` (see graph below):
```graphql
query TotalCountOfCommentsWithEdges {
  posts {
    commentsConnection {
      totalCount
      edges {
        node {
          id
        }
      }
    }
  }
}
```

**Screenshots**
![Screen Shot 2021-07-18 at 12 33 21 AM](https://user-images.githubusercontent.com/10360334/126059461-b73cd607-598a-441b-a9a8-5440b4aea7f9.png)

**System (please complete the following information):**

-   OS: macOS 11.4
-   Version: `@neo4j/graphql@2.0.0-alpha.3`
-   Node.js version: `14.16.0`

**Comment by litewarp** [comment]
So I just submitted a PR, but for some context, I never anticipated that someone would only want the totalCount when calling a connection, so the cypher was not written with that option in mind. 

An easy work-around is in the PR, but if the decision is made to rewrite the cypher so that the database only returns totalCount (instead of returning edges + totalCount to the server), it may require help from someone with more cypher experience.

**Comment by litewarp** [comment]
Nevermind. I think I fixed the cypher so #333 should now fully resolve the issue.

**Comment by darrellwarde** [comment]
The fix for this will be released in 2.0.0-alpha.4 shortly, please let us know if it works for you! 🙂
